cfme/cloud/provider/__init__.py

```diff
@@ -175,20 +176,6 @@ def _form_mapping(self, create=None, **kwargs):
         return {'name_text': kwargs.get('name')}
 
 
-@navigator.register(Provider, 'All')
-class All(CFMENavigateStep):
-    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
-
-    def step(self):
-        from cfme.web_ui.menu import nav
-        nav._nav_to_fn('Compute', 'Clouds', 'Providers')(None)
-
-    def resetter(self):
-        tb.select('Grid View')
-        sel.check(paginator.check_all())
-        sel.uncheck(paginator.check_all())
-
-
 @navigator.register(Provider, 'Add')
 class New(CFMENavigateStep):
     prerequisite = NavigateToSibling('All')
```

Comment:
that's why this one is a wip, i haven't done the transitional navigation's yet

my plan is to start with one step, get feedback, then transition more